debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/ConnectionIT.java

```diff
@@ -40,8 +40,8 @@ public void setUp() {
     @Test
     public void shouldCreateMovieDatabase() {
         useConfiguration(config.edit()
-                .with(MongoDbConnectorConfig.DATABASE_WHITELIST, "dbA,dbB")
-                .with(MongoDbConnectorConfig.COLLECTION_BLACKLIST, "dbB.moviesB")
+                .with(MongoDbConnectorConfig.DATABASE_INCLUDE_LIST, "dbA,dbB")
```

Comment:
Should it use `Configurator.withXXX` method?

Reply:
I didn't change the code, just updated the const name. Maybe log an easy-starter issue to change this everywhere? Do we want to get rid of the "Config-Editor"?